homeassistant/bootstrap.py

```diff
@@ -26,6 +26,7 @@
 # hass.data key for logging information.
 DATA_LOGGING = 'logging'
 
+DEBUGGER_INTEGRATIONS = ('ptvsd',)
```

Comment:
All others are sets, although it's probably faster as a tuple for small collections.

Reply:
CORE_INTEGRATIONS are a tuple, LOGGING and STAGE_1 are a set. No reason why they can't all be tuples really rather than sets. But it's all splitting hairs really and doesn't really matter.

Reply:
Changed it to use set arithmetic anyhow, just to be consistent.